rs/tests/networking/subnet_update_workload/src/lib.rs

```diff
@@ -62,17 +63,19 @@ pub fn setup(
     env: TestEnv,
     nodes_app_subnet: usize,
     boot_image_minimal_size_gibibytes: Option<ImageSizeGiB>,
+    required_host_features: Vec<HostFeature>,
 ) {
     let logger = env.logger();
     PrometheusVm::default()
```

Comment:
I think you also want to allocate the PrometheusVm in the Performance DC like:
```suggestion
PrometheusVm::default()
    .with_required_host_features(required_host_features)
```